.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -799,7 +785,7 @@
       <a href="${asset.path}/index">${_EscapeTool.xml($asset.metadata.displayName)}</a>
   
       #if ( $visibleChildren.size() > 0 )
-        <span class="toggle"><span>+</span></span>
+        #buildToggleIcon
```

Comment:
I think we should always include () even though not needed:

    #buildToggleIcon()

Scans more effortlessly. Without them, easy to confuse with a comment.